cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/RegulatingControlMappingForStaticVarCompensators.java

```diff
@@ -0,0 +1,162 @@
+        mapping = new HashMap<>();
+    }
+
+    public static void initialize(StaticVarCompensatorAdder adder) {
+        adder.setVoltageSetPoint(Double.NaN);
+        adder.setReactivePowerSetPoint(Double.NaN);
+        adder.setRegulationMode(StaticVarCompensator.RegulationMode.OFF);
+    }
+
+    public void add(String iidmId, PropertyBag sm) {
+        String rcId = parent.getRegulatingControlId(sm);
+        boolean controlEnabledProperty = sm.asBoolean("controlEnabled", false);
+        double defaultTargetVoltage = sm.asDouble("voltageSetPoint");
+        double defaultTargetReactivePower = sm.asDouble("q");
+        String defaultRegulationMode = sm.getId("controlMode");
+
+        if (mapping.containsKey(iidmId)) {
+            throw new CgmesModelException(
+                "StaticVarCompensator already added, IIDM StaticVarCompensator Id : " + iidmId);
+        }
+
+        RegulatingControlForStaticVarCompensator rc = new RegulatingControlForStaticVarCompensator();
+        rc.regulatingControlId = rcId;
+        rc.controlEnabledProperty = controlEnabledProperty;
+        rc.defaultTargetVoltage = defaultTargetVoltage;
+        rc.defaultTargetReactivePower = defaultTargetReactivePower;
+        rc.defaultRegulationMode = defaultRegulationMode;
+
+        mapping.put(iidmId, rc);
+    }
+
+    void applyRegulatingControls(Network network) {
+        network.getStaticVarCompensatorStream().forEach(this::apply);
+    }
+
+    private void apply(StaticVarCompensator svc) {
+        RegulatingControlForStaticVarCompensator rd = mapping.get(svc.getId());
+        apply(svc, rd);
+    }
+
+    private void apply(StaticVarCompensator svc, RegulatingControlForStaticVarCompensator rc) {
+        if (rc == null) {
+            return;
+        }
+        if (!rc.controlEnabledProperty) {
+            return;
+        }
+
+        String controlId = rc.regulatingControlId;
+        if (controlId == null) {
+            context.missing("Regulating control Id not defined");
+            setDefaultRegulatingControl(rc, svc);
+            return;
+        }
+
+        RegulatingControl control = parent.cachedRegulatingControls().get(controlId);
+        if (control == null) {
+            context.missing(String.format("Regulating control %s", controlId));
+            setDefaultRegulatingControl(rc, svc);
+            return;
+        }
+        if (!control.enabled) {
+            return;
+        }
+
+        if (context.terminalMapping().areAssociated(control.cgmesTerminal, control.topologicalNode)) {
+            setRegulatingControl(control, svc);
+        } else {
+            context.pending(
+                String.format(
+                    "Remote regulating control for static var compensator %s replaced by control defined at Equipment",
+                    svc.getId()),
+                "IIDM model does not support remote control for static var compensators");
+
+            setDefaultRegulatingControl(rc, svc);
+        }
```

Comment:
I think you are changing the previous behaviour: in case of remote control, we used to set the regulation as voltage local. Here, it seems you are using the one defined at Equipment which could be consistent with the remote regulating control. Is it really what we want? Shouldn't it only be regulated locally in this case?

Reply:
I did not change the previous behaviour. I only changed the warning message becasuse was incoherent with the code implementation. If there is not terminal mapping a `defaultRegulatingControl  `is defined and as `voltageSetPoint `the voltage setpoint of the `staticVarCompensator `is set (not the nominal voltage)

Reply:
Okay, I was indeed confused by the previous warning message which was misleading, my bad. Thanks for the change.